raiden/transfer/mediated_transfer/events.py

```diff
@@ -74,7 +74,10 @@ def to_dict(self) -> Dict[str, Any]:
         result = {
             "payment_id": str(self.payment_id),
             "message_id": str(self.message_id),
-            "message_order": str(self.message_order)
+            "message_order": str(self.message_order),
+            "message": str(self.message),
```

Comment:
For the message field, since it is an istance of the `Message ` class we should use the `to_dict `hook for serialization